builtin/logical/mysql/path_config_connection.go

```diff
@@ -17,12 +18,6 @@ func pathConfigConnection(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: "DB connection string",
 			},
```

Comment:
Don't get rid of this. We'll have something newer soon enough anyways, and there's no point potentially breaking backwards compatibility.